.github/workflows/build-install-test-snap.yaml

```diff
@@ -12,6 +12,9 @@ on:
       snapcraft-args:
         required: false
         type: string
+      snap-install-args:
+        required: false
+        type: string
```

Comment:
For reference, [`"If a default parameter is not set, the default value of the input is [...] "" for a string."`](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onworkflow_callinputs)

Reply:
mm yes, that is actually an issue because with ```sudo snap install ros2-talker-listener_0.1_amd64.snap ""``` it will return the error: cannot install snap with empty name

Reply:
added an extra condition to solve this